test/python/transpiler/test_coupling.py

```diff
@@ -436,3 +439,9 @@ def test_subgraph(self):
         edge_list = subgraph.get_edges()
         expected = [(0, 1), (1, 2), (2, 3)]
         self.assertEqual(expected, edge_list, f"{edge_list} does not match {expected}")
+
+    @unittest.skipUnless(optionals.HAS_GRAPHVIZ, "Graphviz not installed")
+    def test_coupling_draw(self):
+        """Test that the coupling map can be drawn."""
+        cmap = CouplingMap([[0, 1], [1, 2], [2, 3], [2, 4], [2, 5], [2, 6]])
+        cmap.draw()
```

Comment:
Sorry I missed your earlier question. It'd be good if we split this out into a dedicated test class so we can use the image test base class. Then we can assert the output looks like what's expected. Right now all this does is test that creating a coupling map and calling `draw()` doesn't error and it would have passed prior to your fix.